enterprise/frontend/src/metabase-enterprise/embedding/components/EmbeddingAppSameSiteCookieDescription/SameSiteSelectWidget.tsx

```diff
@@ -1,8 +1,28 @@
 import { useState } from "react";
+import { t } from "ttag";
 
 import { Button, Group, Icon, Menu, Text } from "metabase/ui";
 import type { SessionCookieSameSite } from "metabase-types/api";
 
+const SAME_SITE_OPTIONS: Options[] = [
```

Comment:
move values from props in `selectors.js`. To make this component less dependent on props.